app/src/main/java/org/mozilla/tv/firefox/webrender/CustomContentRequestInterceptor.kt

```diff
@@ -43,7 +43,10 @@ class CustomContentRequestInterceptor(
             URLs.URL_LICENSES -> {
                 // Prevent getting stuck in this loop when clicking back from the activity
                 Handler(Looper.getMainLooper()).post { context.serviceLocator.sessionRepo.attemptBack() }
-                startActivity(context, Intent(context, OssLicensesMenuActivity::class.java), null)
+                val intent = Intent(context, OssLicensesMenuActivity::class.java).apply {
+                    flags = Intent.FLAG_ACTIVITY_NEW_TASK
```

Comment:
FYI: I noticed some strange behaviors after this patch and filed a new issue: https://github.com/mozilla-mobile/firefox-tv/issues/2963

I'm guessing these are present in the original implementation.